common/lib/dependabot/errors.rb

```diff
@@ -223,6 +223,15 @@ def self.updater_error_details(error)
           "go-mod": error.go_mod
         }
       }
+    when
+      DependencyConflict,
```

Comment:
For `DependencyConflict` we already have a more generic variant: `Dependabot::DependencyFileNotResolvable`.

In the Go ecosystem we use this for any issue when resolving dependencies: https://github.com/dependabot/dependabot-core/blob/569d4ef81b0cd892de7867c5688059284d2c96aa/go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb#L19-L35

If you want to proceed with `DependencyConflict` instead that's ok with me. Just wanted to mention it.

Reply:
Going to replace it then.

Reply:
replaced the DependencyConflict with DependencyFileNotResolvable